### PR TITLE
Integrated Thea Council 

### PR DESCRIPTION
**Pull request by zktony**
Governance Pallet on Parachain:-
* Add a new member. (the bulk of current members or sudo user).
* Remove existing members. (majority of current members or sudo user).
* Delete malicious withdrawal requests. (Any member can do it).

XCM Handler Pallet:-
* Added withdrawal delay.
* Withdraw execution inside on_init function.

Will close https://github.com/Polkadex-Substrate/thea/issues/78 & https://github.com/Polkadex-Substrate/thea/issues/74

### CODE REVIEW

**Comment by serhii-temchenko** [comment]
I it ready for review? If so - please add appropriate label

File: pallets/thea-council/src/tests.rs [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+// #[test]
+// fn correct_error_for_none_value() {
+//     new_test_ext().execute_with(|| {
+//         // Ensure the expected error is thrown when no value is present.
+//         assert_noop!(
+// 			TemplateModule::cause_error(RuntimeOrigin::signed(1)),
+// 			Error::<Test>::NoneValue
+// 		);
+//     });
+// }
```
Comment (serhii-temchenko) [inline pallets/thea-council/src/tests.rs:149]: What this about?

**Review by serhii-temchenko — CHANGES_REQUESTED** [review]

File: pallets/thea-council/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+		TransactionDeleted(u32),
+	}
+
+	// Errors inform users that something went wrong.
+	#[pallet::error]
+	pub enum Error<T> {
+		/// Error names should be descriptive.
+		NoneValue,
+		/// Errors should have helpful documentation associated with them.
+		StorageOverflow,
```
[the remark this answers is not in the document]
Comment (zktony) [reply pallets/thea-council/src/lib.rs]: done

File: pallets/thea-council/src/tests.rs (hunk 1 above)
Comment (zktony) [reply pallets/thea-council/src/tests.rs]: removed

**Review by serhii-temchenko — APPROVED** [review]

**Review by Gauthamastro — APPROVED** [review]

**Pull request merged**